src/billing/api/index.ts

```diff
@@ -84,6 +87,40 @@ export const getBillingCreditCardParams = (): ReturnType<typeof getPaymentForm>
   return makeResponse(200, cc, 'getBillingCreditCard')
 }
 
+export const getOrgsLimits = (): ReturnType<typeof apiGetOrgLimits> => {
+  const limits: OrgLimits = {
+    orgID: 'org123',
+    rate: {
+      readKBs: 100,
+      writeKBs: 100,
+      cardinality: 1,
+    },
+    bucket: {
+      maxRetentionDuration: 3,
+      maxBuckets: 2,
+    },
+    task: {
```

Comment:
It's so weird that we have an object just to contain One property. Why can't it be just `maxTasks` or maybe a key which has something like this:
```
maximums: {
    tasks: 10,
    dashboards: 7,
    checks: 4,
    ....
}
```

Reply:
Agreed. I think it's a construct of the API contracts that current exist in Quartz and IDPE.